packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -303,6 +318,23 @@ class Auth {
         } else {
             dntCookieStatus = Boolean(Number(dntCookieVal))
         }
+
+        if (options?.includeDefaults) {
+            const defaultDnt = this.defaultDnt
+
+            let finalDntValue
+            const dntCookie = dntCookieVal === '1' ? true : dntCookieVal === '0' ? false : undefined
+            if (dntCookie !== undefined) {
+                finalDntValue = dntCookie
+            } else {
+                // If the cookie is not set, read the defaultDnt preference.
+                // If defaultDnt doesn't exist, default to false, following SLAS default for dnt
+                finalDntValue = defaultDnt !== undefined ? defaultDnt : false
```

Comment:
@jeremy-jung1 by default, ECOM will track the data. (dnt=false). I think we want to set this to be `true` by default, meaning we are not tracking the users unless they said so.

Reply:
I got a bit confused reading this function since I saw that we are returning either dntCookieStatus or finalDntValue. This function might need some clarification.

I think the intent of this function is to return the value of the DNT cookie or undefined if the cookie doesn't exist / there is a mismatch between the cookie and access token?

Or is it meant to return the actual DNT value used?

Reply:
Yes this function is to return two things regarding the state of DNT:
1. the DNT cookie, which represents user preference. This is used to determine whether or not to show the consent tracking form
2. the final DNT value (the actual DNT value). This is used to incorporate DNT in the analytics layers.

Following some feedback, I made updates to the names of the variables to be clearer:
dntStatus -> selectedDoNotTrackValue
finalDntValue -> effectiveDoNotTrackValue